gluten-core/src/main/scala/io/glutenproject/expression/PredicateExpressionTransformer.scala

```diff
@@ -58,7 +58,9 @@ object InExpressionTransformer {
       values: Set[Any],
       valueType: DataType): ExpressionNode = {
     val expressionNodes = new java.util.ArrayList[ExpressionNode](
-      values
+      values.toSeq
```

Comment:
I note this method is shared by `In/InSet`. But the purpose is to only fix `InSet`. Should we keep no change for `In`?

Reply:
Well, for now, our goal is to maintain consistent behavior, and the Gluten In Function is also converted by Set, so it is acceptable for changes to involve In Function.